aiorussound/const.py

```diff
@@ -154,7 +156,19 @@ class FeatureFlag(Enum):
 
 VERSIONS_BY_FLAGS = defaultdict(list)
 
-ZONE_PROPERTIES: list[str] = ["currentSource"]
+ZONE_PROPERTIES: list[str] = [
+    "currentSource",
+    "favorite[1].valid",
```

Comment:
Why are we loading these values into the cache on connection? The enumerate command will typically fetch these values directly instead of grabbing them from the cache.

How we handle this is completely up for debate, just trying to understand it better.

Reply:
No compelling reason other than it would happen during startup regardless.  Since this is dependent on controller type, it doesn't make sense to pre-fetch these.